### PR TITLE
i18n: extract strings

### PR DESCRIPTION
**Pull request by alalamav**
* First i18n pass: integrates Polymer's app-localize-behavior to support l10n.
* Extracts EN strings and loads them dynamically. 
* TODOs:
  * Detect language, allow the user to change it.
  * Import translations.
  * RTL mirroring.
  * Minor fixes, marked as TODOs in the code.

### CODE REVIEW

File: src/server_manager/ui_components/app-root.html [hunk 1]
```diff
@@ -497,6 +510,11 @@ <h4>Disconnect DigitalOcean account</h4>
         'SetUpGcpRequested': 'handleSetUpGcpRequested',
         'ManualServerEntryCancelled': 'handleManualCancelled'
       },
+      ready: function() {
+        // TODO(alalama): detect language
+        const messagesUrl = `/messages/${this.language}.json`;
+        this.loadResources(messagesUrl, this.language);
```
Comment (fortuna) [inline src/server_manager/ui_components/app-root.html]: Do we need this? Aren't we passing the localize function?

File: src/server_manager/model/server.ts [hunk 2]
```diff
@@ -120,7 +120,7 @@ export interface ManagedServerRepository {
   getRegionMap(): Promise<Readonly<RegionMap>>;
   // Creates a server and returning it when it becomes active (i.e. the server has
   // created, not necessarily once shadowbox installation has finished).
-  createServer(region: RegionId): Promise<ManagedServer>;
+  createServer(region: RegionId, name: string): Promise<ManagedServer>;
```
Comment (fortuna) [inline src/server_manager/model/server.ts]: I don't see this function called in your PR. Shouldn't you change the call site?

File: src/server_manager/ui_components/app-root.html (hunk 1 above)
Comment (alalamav) [reply src/server_manager/ui_components/app-root.html]: We do - this is the function that loads the strings for `localize`.

File: src/server_manager/model/server.ts (hunk 2 above)
Comment (alalamav) [reply src/server_manager/model/server.ts]: It's [here](https://github.com/Jigsaw-Code/outline-server/pull/446/files#diff-e3238a12d6d4f2299200994f4565a21fR726). (The file diff was collapsed).

**Comment by alalamav** [comment]
That's a fair point. Static checks would work only for localization calls in TS, since the UI components call it from HTML and JS. Most of the calls happen in this context.

I'll think about solutions, but it seems that the lack of type safety in the UI components is the real issue.

**Comment by fortuna** [comment]
Another thing that is unclear to me: What happens if a feature doesn't have translations yet? Does it show empty strings? English strings? ids?
How do we know when a feature is translated?

**Comment by alalamav** [comment]
We have generally released features only after localization of the strings have been completed. In the Outline clients we have a [script](https://github.com/Jigsaw-Code/outline-client/blob/master/scripts/l10n/validate_localized_keys.py) that validates that all the keys are present in all languages. 
The app-localize-behavior supports a property (`useKeyIfMissing`) to display the key name when the value is missing from resources.

**Pull request merged**